plugins/hls-alternate-number-format-plugin/test/Main.hs

```diff
@@ -0,0 +1,88 @@
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE TypeOperators  #-}
+{-# LANGUAGE ViewPatterns   #-}
+module Main ( main ) where
+
+import           Data.List                        (find)
+import           Data.Text                        (Text)
+import qualified Data.Text                        as T
+import qualified Ide.Plugin.AlternateNumberFormat as AlternateNumberFormat
+import qualified Ide.Plugin.Conversion            as Conversion
+import           System.FilePath                  ((</>))
+import           Test.Hls
+import           Text.Regex.TDFA                  ((=~))
+
+main :: IO ()
+main = defaultTestRunner test
+
+alternateNumberFormatPlugin :: PluginDescriptor IdeState
+alternateNumberFormatPlugin = AlternateNumberFormat.descriptor "alternateNumberFormat"
+
+test :: TestTree
+test = testGroup "alternateNumberFormat" [
+    codeActionHex "TIntDtoH" 3 13]
```

Comment:
you should run `stylish-haskell` on your codebase.

If this is already `stylish-haskell` formatted, ignore this comment.

Reply:
Yeah, I normally use Brittany, and for whatever reason `stylish-haskell` seems to just accept this?